packages/commonwealth/server/routes/getThreads.ts

```diff
@@ -40,13 +41,19 @@ const getThreads = async (models: DB, req: Request, res: Response) => {
         },
       ],
     });
+
+    threads = await getThreadsWithCommentCount({
+      threads: threads.map((th) => th.toJSON()),
+      models,
+      chainId: chain?.id,
```

Comment:
change to 
`chainId: chain?.id || threads[0].chain,` to satisfy this https://github.com/hicommonwealth/commonwealth/pull/3959#issuecomment-1562242248

Reply:
no sure if it will work when thread ids from different communities are passed in query params - i think it won't ideally happen tho.

Reply:
i don't thin we allow it intentionally though
- to be safe you can add for-loop - for all different chain_ids!
- but would be a overkill i believe!